src/lua/protobuf_wireformat.lua

```diff
@@ -97,6 +141,29 @@ local function encode_varint(value)
     return ffi.string(buf, size)
 end
 
+-- Decode input value as VARINT without a tag.
+--
+-- Input value type: string
```

Comment:
It looks like the function accepts the decoding context, not a string. There is a string in `ctx.val`, but `val.pos` is also the input parameter.

Some functions says that the input value is `number` and it is even more confusing.

What is the idea behind this remark? Does it add useful information for a reader?

Reply:
I agree that it may be confusing. I will replace input value type with 'decoding context' where `ctx` passed as parameter.

Reply:
Seems solved now.